gotfparse/pkg/converter/converter.go

```diff
@@ -284,7 +284,7 @@ func (t *terraformConverter) getAttributeValue(a *terraform.Attribute) any {
 
 	// Only attempt to handle functions manually if the value is null or not known
 	// This ensures we don't interfere with functions that have been successfully resolved
-	if val.IsNull() || !val.IsKnown() {
+	if val.IsNull() || !val.IsKnown() || !val.IsWhollyKnown() {
```

Comment:
Because we continue to dance with this idea of how much value to wring out of partially known values, I do wonder if folks will want to make assertions about dynamic blocks in cases where some of the elements are known and some aren't :thinking: . This is definitely the safer/simpler case though, so let's roll with it until we have a good reason to make it more complex :grin: .